src/app.module.ts

```diff
@@ -16,7 +16,7 @@ import { UserSession } from "src/core/infrastructure/entities/Auth";
 			username: "postgres",
 			password: "password",
 			database: "tribal",
-			entities: [User, UserSession],
+			entities: [UserRecord, UserSessionRecord],
```

Comment:
I've renamed these to `XRecord` so we know they're referring to database records. 

We need to include these in the entities here so we can use them in our APP